README.md

```diff
@@ -59,6 +59,7 @@ These are the main features:
     - [Default settings](#default-settings)
     - [Placeholders](#placeholders)
     - [Environment variables](#environment-variables)
+    - [Controlling what will be exposed on the client](#controlling-what-will-be-exposed-on-the-client)
```

Comment:
With the whitelist this is more about `Exposing config values to the client` rather than `Controlling what will be exposed…`, because by default there's nothing exposed besides the `basePath`. So this whole topic isn't about taking care not to expose secrets accidentally anymore, but rather to expose explicitly, what can be exposed.